src/views/Auth/Login.test.jsx

```diff
@@ -34,13 +34,13 @@ it('should allow the user to log in', async () => {
           </Route>
         </Switch>
       </MemoryRouter>
-    </ProvideAuth>
+    </ProvideAuth> 
   );
 
   expect(container).toMatchSnapshot();
 
-  const emailField = screen.getByLabelText('Email');
-  const passwordField = screen.getByLabelText('Password');
+  const emailField = screen.getByLabelText(/email/i);
```

Comment:
These two formats are actually equivalent. Your update simply changes the lookup to use RegEx. :)